File: 9-idade em dias/idade.c
#include <stdio.h>

int main() {
    int anos, meses, dias, totalDias;
    printf("Digite sua idade em anos, meses e dias: ");
    scanf("%d %d %d", &anos, &meses, &dias);

    totalDias = (anos * 365) + (meses * 30) + dias;
    printf("Sua idade em dias é: %d\n", totalDias);
    return 0;
}

File: 9-idade em dias/idade.go
package main

import "fmt"

func main() {
	var anos, meses, dias int
	fmt.Println("Digite sua idade em anos, meses e dias: ")
	fmt.Scan(&anos, &meses, &dias)

	totalDias := (anos * 365) + (meses * 30) + dias
	fmt.Printf("Sua idade em dias é: %d\n", totalDias)
}
